packaging/rpm/kepler.spec

```diff
@@ -33,7 +33,7 @@ CROSS_BUILD_BINDIR=_output/bin
 GOARCH=amd64
 %endif
 
-make _build_local GOOS=${GOOS} GOARCH=${GOARCH}
+make _build_local GOOS=${GOOS} GOARCH=${GOARCH} ATTACHER_TAG=libbpf
```

Comment:
instead, we make the switch in the `Makefile` itself to use `libbpf` as the default attacher?

Reply:
perhaps in next PR? lets this change get settled?

Reply:
works for me :-)